index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo">Dhuruv</div>
+        <div>
+            <img class="nav-icon" src="/Icons/Enabled.png" alt="nav button">
+        </div>
+    </header>
+    <main>
+        <section>
+            <h1>
```

Comment:
- [ ] [OPIONAL] you should use `Crete Round` as font not `Poppins` please try to use it here 
example: 
```CSS
font-family: 'Crete Round';
```

Reply:
ok